### ISSUE TITLE
fv3 needs to export the 1-ice fraction that it used in calculating fluxes for export to the coupled model

### ISSUE THREAD

**Issue by DeniseWorthen** [NOAA-EMC/ufsatm#230]
## Description

FV3 needs to export the open-water fraction to CMEPS so that CMEPS can properly normalize the fluxes imported from FV3 which are sent to other components (ocean,ice).

## Solution
A new export variable can be added in setup_exportdata

**Comment by junwang-noaa** [comment]
Denise, do you have a standard field name for this open water fraction field? I will use "open_water_faction" for now. I think the same name needs to be used in nems field dictionary and CMEPS.

**Comment by DeniseWorthen** [comment]
Yes, it will need to be the same. My working branch is [here](https://github.com/DeniseWorthen/fv3atm/tree/feature/sa_ofrac). I've called it openwater_frac_in_atm. This will be aliased in CMEPS to the name "Sa_ofrac".

This branch also contains a minor bugfix for when dumpfields=true.

**Comment by junwang-noaa** [comment]
Your code changes look good. So in the coupled mode, since fv3 sends the
available fields to mediator before model start integration, the open water
frac in this case is from atmos ICs, it may not be consistent with the ice
fraction from the ice model, will it cause some issue in mediator?

**Comment by DeniseWorthen** [comment]
Thats a good point but I haven't seen any differences in the exchanged fields until after a few timesteps. The field that FV3 is sending back is meant to be identical to the field it received. Implementing it this way is much simpler than having CMEPS keep two time-levels.

**Comment by SMoorthi-emc** [comment]
Why do you need this?
Open_water=(1-ice_frac)*ocean_frac.
Moorthi

**Comment by DeniseWorthen** [comment]
because cmeps needs the open water fraction to re-normalize and it no longer has that value available since it would need to be the previous timestep.

**Comment by junwang-noaa** [comment]
@DeniseWorthen Is there any update on committing the code changes?

**Comment by DeniseWorthen** [comment]
I was going to update the ufs-weather issue [#366](https://github.com/ufs-community/ufs-weather-model/issues/366) today. Yesterday we got Bob Oehmke from ESMF involved in the ancillary issue which arose as part of adding this feature to try to determine if that ancillary issue is an ESMF or CMEPS bug.

I've tested the export field I've created for the open water fraction and it is nearly identical to the imported open water fraction from at the previous timestep. This is the expected behaviour. 

As long as this field is not connected, I believe the change could be added to FV3 at any time with no impact on baselines. The FV3 branch I am using for adding this feature is [here](https://github.com/DeniseWorthen/fv3atm/tree/feature/sa_ofrac).
